Sources/Paywall/Network/PaywallResponseManager.swift

```diff
@@ -23,18 +23,51 @@ class PaywallResponseManager: NSObject {
 	}
 	
 	func getResponse(identifier: String? = nil, event: EventData? = nil, completion: @escaping (PaywallResponse?, NSError?) -> ()) {
-		
+        var experimentId: String?  = nil
+        var variantId: String? = nil
+        var identifier = identifier;
+
 		// if we're requesting a response for a trigger/event that wasn't in the initial config endpoint, ignore it
 		if Paywall.shared.didFetchConfig, let eventName = event?.name {
-			if !Store.shared.triggers.contains(eventName) {
-				// create the error
-				let userInfo: [String : Any] = [
-					NSLocalizedDescriptionKey :  NSLocalizedString("Trigger Disabled", value: "There isn't a paywall configured to show in this context", comment: "") ,
-				]
-				let error = NSError(domain: "SWTriggerDisabled", code: 404, userInfo: userInfo)
-				completion(nil, error)
-				return
-			}
+            let triggerResponse = TriggerManager.shared.handleEvent(eventName: eventName)
+            switch(triggerResponse) {
+                // Do nothing, continue with loading as expected
+            case .PresentV1:
+                    break;
+                
+            case .PresentIdentifier(let _experimentId,  let _variantId, let paywallIdentifier):
+                identifier = paywallIdentifier;
+                experimentId = _experimentId
+                variantId = _variantId
+                break;
+            case .Holdout(let experimentId, let variantId):
+                let userInfo: [String : Any] = [
```

Comment:
I think we should change how the error code & domains work here. Domain should be something consistent everywhere in our SDK but code can change. Not sure if we'd be breaking anything by changing the existing trigger logic to match that.